views/Home.jsx

```diff
@@ -1,14 +1,17 @@
 const React = require('react');
+const Layout = require('./Layout');
 
 function Home() {
-
   return (
-    <main>
-      <h1>Home</h1>
-    </main>
+    <Layout>
+      <main>
+        <h1>Home</h1>
+        <img src="/public/images/beer.png" alt="" />
```

Comment:
Image is not correctly linked, I can't see it. 

You should omit the `/public` part, it's not necessary from now on. :D